### DISCUSSION TITLE
Neumorphism UI Dark For Flet

### DISCUSSION THREAD

**Discussion by 3booodpro** [flet-dev/flet#5667] [Extensions]
I think Flet is really amazing and very productive!!

but the issue here its not so customizable..
like I wanted to add Shadows to a Container but I couldn't
additionally blurring the container isn't supported

why would anyone ask for them?
short answer : making the app (or the web) look good! I wanted to achieve Neumorphism UI Dark with flet but I wasn't able to.. because Flet lacks extra customization..

it would be great to consider them in the future! thx!

**Comment by 3booodpro** [comment]
![image](https://user-images.githubusercontent.com/78119395/212498456-077df214-4716-40ac-80f4-aa0482716e03.png)
Here's what I mean with extra customization! to achieve something like this!

**Comment by JorgeAlvarez21** [comment]
The only way i was able to use shadows was using an elevated button instead of container and put everything inside it. You can set its state to disable if you don't care for the hover effect. if you want to see it is in my repo named 'Fletdev_Mobile_App_Tracker'.
I agree with you %100, these features would be great to have as they can make the UI look pretty sleek. Maybe we can request them, shouldn't be too hard to have them implemented.

Your app looks super cool btw! I'll be following you to see more of it!

**Comment by ndonkoHenri** [comment]
What do you guys think of this dart package? 
https://pub.dev/packages/flutter_neumorphic

**Comment by FeodorFitsner** [comment]
Look like rubber buttons on TV remote control :)

**Comment by Haadi-A** [comment]
> Look like rubber buttons on TV remote control :)

Yeah but light mode neumorphic designs looks great. Like this

![another neumorphic](https://user-images.githubusercontent.com/77079200/216723721-91cc0438-8ad6-4afa-a5d2-93e79fd366b6.jpeg)

**Comment by FeodorFitsner** [comment]
I think this kind of functionality should be implemented as a separate Flutter package (plus Python package) that could extend a standard Flet client.

**Comment by ndonkoHenri** [comment]
Package to be implemented: https://pub.dev/packages/flutter_neumorphic_plus
